documentation/use-cases/Story1-UseCaseNarrative.md

```diff
@@ -0,0 +1,43 @@
+# STORY 1 - USE CASE NARRATIVE
+
+<b>Use case</b>: Check children's marks 
+
+<b>Scope</b>:  /   
+
+<b>Level</b>: User-goal  
+
+<b>Intention in context</b>: Parent wants to know the marks of his/her children
+
+<b>Primary actor</b>: Parent  
+
+<b>Secondary actor(s)</b>:  /  
+
+<b>Stakeholders' interests</b>:  
+	
+* Parent: would like to see the marks in a few steps
+* Principal: would like parents to know the scholastic trend of their children
+
+<b>Precondition</b>:
+
+* Parent should be logged in
+
+<b>Minimum guarantees</b>:  /  
+
+<b>Success guarantees</b>: Parent knows the marks of his/her children 
+
+<b>Triggers</b>:  /  
+
+<b>Main Success Scenario</b>:  
+
+1. The parent selects the child
+2. The system shows available subjects
+3. The parent selects the subject
+4. The system shows the marks of the selected subject and the date
+
+The use case terminates with success.  
+
+<b>Extensions</b>:  /

```

Comment:
Forse potrebbe esserci un alternative success path: anziché per materia il genitore potrebbe vedere i voti del figlio per data: in questo caso lo step 3 sarebbe scegliere un range di date corretto (comunque si tratta di un trend)

Reply:
Si solo questo cambierei

Reply:
Ma lo mettereste nelle extensions o solo come ulteriore step?